tests/test_models/test_backbones/test_shufflenet_v1.py

```diff
@@ -0,0 +1,335 @@
+import pytest
+import torch
+from mmcv.utils import is_tuple_of
+from torch.nn.modules import GroupNorm
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmseg.models.backbones import ShuffleNetV1
+from mmseg.models.backbones.shufflenet_v1 import ShuffleUnit
+
+
+def is_block(modules):
+    """Check if is ResNet building block."""
+    if isinstance(modules, (ShuffleUnit, )):
+        return True
+    return False
+
+
+def is_norm(modules):
+    """Check if is one of the norms."""
+    if isinstance(modules, (GroupNorm, _BatchNorm)):
+        return True
+    return False
+
+
+def check_norm_state(modules, train_state):
+    """Check if norm layer is in correct train state."""
+    for mod in modules:
+        if isinstance(mod, _BatchNorm):
+            if mod.training != train_state:
+                return False
+    return True
```

Comment:
These are in mmcv

Reply:
In `tests/test_models/test_backbones/utils.py`.
The redundant code will be removed.